Tests fail due to missing dependency on imageio

Test currently fails with:

```
typhon/__init__.py:18: in <module>
    from . import geographical
typhon/geographical.py:7: in <module>
    import imageio
E   ModuleNotFoundError: No module named 'imageio'
```

`typhon/geographical.py` and `typhon/retrieval/spareice/common.py` import imageio which is not a dependency of typhon yet. Should we do a late import at the two places where it is used or make it a hard dependency?

--- a/setup.py
+++ b/setup.py
@@ -92,6 +92,7 @@ setup(
     # https://packaging.python.org/en/latest/requirements.html
     install_requires=[
         'docutils',
+        'imageio',
         'matplotlib>=1.4',
         'netCDF4>=1.1.1',
         'numba',

